repos/system_upgrade/el7toel8/actors/pcidevicesscanner/tests/test_pcidevicesscanner.py

```diff
@@ -45,10 +69,9 @@ def test_parse_pci_devices(current_actor_libraries):
     assert dev.dev_cls == 'IDE interface'
     assert dev.vendor == 'Intel Corporation'
     assert dev.name == '82371SB PIIX3 IDE [Natoma/Triton II]'
-    assert dev.subsystem_vendor == 'Red Hat, Inc.'
-    assert dev.subsystem_name == 'Qemu virtual machine'
     assert dev.progif == '80'
     assert dev.driver == 'ata_piix'
+    assert dev.pci_id == "15b43:0722"
```

Comment:
@ZhukovGreen  and please fix all quotes (replace by apostrophes) as well. Mixing quotes and apostrophes in this way is bad practice. As postrophes are default here as I see, use the apostrophes.

Reply:
Yes, I did a mistake, thx for catchin this. Will be fixed in the next commit